kubernetes/updater/cmd/teleport-kube-updater/main.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+	"time"
+
+	"github.com/docker/distribution/reference"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	runtimescheme "sigs.k8s.io/controller-runtime/pkg/scheme"
+
+	"github.com/gravitational/teleport/kubernetes/updater/pkg/controller"
+	"github.com/gravitational/teleport/kubernetes/updater/pkg/img"
+	"github.com/gravitational/teleport/kubernetes/updater/pkg/maintenance"
+	"github.com/gravitational/teleport/kubernetes/updater/pkg/version"
+)
+
+var (
+	SchemeBuilder = &runtimescheme.Builder{GroupVersion: appsv1.SchemeGroupVersion}
+	scheme        = runtime.NewScheme()
+)
+
+const (
+	namespace   = "namespace"
+	releaseName = "release"
+)
+
+func init() {
+	SchemeBuilder.Register(
+		&appsv1.Deployment{},
+		&appsv1.DeploymentList{},
+		&appsv1.StatefulSet{},
+		&appsv1.StatefulSetList{},
+	)
+	utilruntime.Must(SchemeBuilder.AddToScheme(scheme))
+}
+
+func main() {
+	ctx := ctrl.SetupSignalHandler()
+
+	var metricsAddr string
+	var probeAddr string
+	var leaderElectionID string
+	var syncPeriodString string
+
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
```suggestion
	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
```
It's easier to write